examples/relative_free_energy.py

```diff
@@ -301,20 +117,9 @@ def read_from_args():
     with open(f"atom_mapping_{args.mol_a_name}_to_{args.mol_b_name}.svg", "w") as fh:
         fh.write(res)
 
-    box_width = 4.0
-    solvent_sys, solvent_conf, solvent_box, solvent_top = builders.build_water_system(box_width)
-    solvent_box += np.diag([0.1, 0.1, 0.1])  # remove any possible clashes
-    solvent_host_config = HostConfig(solvent_sys, solvent_top, solvent_conf, solvent_box)
-
     forcefield = Forcefield.load_from_file(args.forcefield)
 
-    estimate_relative_free_energy(mol_a, mol_b, core, forcefield, solvent_host_config, n_frames=100, prefix="solvent")
-
-    complex_sys, complex_conf, _, _, complex_box, complex_top = builders.build_protein_system(args.protein)
-    complex_box += np.diag([0.1, 0.1, 0.1])  # remove any possible clashes
-    complex_host_config = HostConfig(complex_sys, complex_top, complex_conf, complex_box)
-
-    estimate_relative_free_energy(mol_a, mol_b, core, forcefield, complex_host_config, n_frames=100, prefix="complex")
+    run_pair(mol_a, mol_b, core, forcefield, args.protein)
```

Comment:
Missing `n_frames`?

Reply:
oops - good call (too many scripts to check... will revalidate all the examples once more before final merge)